postalicious/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Postalicious</title>
+  <link rel="stylesheet" href="./styles.css">
```

Comment:
You will almost never want to use relative asset urls. Change this to `<link rel="stylesheet" href="/styles.css">` and ensure all asset urls are absolute AKA start with a `/`